### PR TITLE
Movie Card, Rating, Menu Pop Over Component

### PR DESCRIPTION
**Pull request by auqibyousuf**

### CODE REVIEW

File: app/components/Rating/Rating.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          <sup className='text-[6px] -top-[4px]'>%</sup>
+        </span>
+      </div>
+    </div>
+  )
+}
+
+export default Rating
+
+// border-2 outline outline-black outline-2
```
Comment (UmmarHamid) [inline app/components/Rating/Rating.tsx]: lets remove this

File: app/components/Rating/Rating.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      'border-red-600': value > 0 && value < 59,
+    }
+  )
+  return (
+    <div
+      className={`w-[38px] h-[38px] p-[2px] rounded-full bg-black ${
+        extraClasses ?? ''
+      }`}
+    >
+      <span className={psedoClasses} aria-hidden='true'></span>
```
Comment (UmmarHamid) [inline app/components/Rating/Rating.tsx:26]: Tailwind has something called as ring inbuilt , can you check that instead?

File: app/components/Rating/Rating.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      <div className={`${ratingClasses} `}>
+        <span className='text-[10px] block'>
+          {value}
+          <sup className='text-[6px] -top-[4px]'>%</sup>
```
Comment (UmmarHamid) [inline app/components/Rating/Rating.tsx:30]: would top work here without position?

File: app/components/MovieCard/MovieCard.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  const movieCardClasses = classNames(
+    'relative flex flex-col max-w-[150px] gap-6'
+  )
+
+  return (
+    <div
+      className={`${movieCardClasses} ${extraClasses ?? ''}`}
+      onClick={onCardClick}
+    >
+      <a href='#'>
```
Comment (UmmarHamid) [inline app/components/MovieCard/MovieCard.tsx]: we should make this as the wrapper for the whole component

**Review by UmmarHamid — CHANGES_REQUESTED** [review]
The commit messages are not acceptable, also the changes w.r.t button should have been part of another PR or at-least another commit.
It becomes very hard for anyone to review

File: app/components/Rating/Rating.tsx (hunk 3 above)
Comment (auqibyousuf) [reply app/components/Rating/Rating.tsx:30]: yes its inbuilt property of this elemt

File: app/components/Rating/Rating.tsx (hunk 2 above)
Comment (auqibyousuf) [reply app/components/Rating/Rating.tsx:26]: its box shadow but here i am using different type of styling which needs psuedo classes

**Review by UmmarHamid — APPROVED** [review]

**Pull request merged**